master/internal/api_notebook.go

```diff
@@ -130,10 +130,13 @@ func (a *apiServer) IdleNotebook(
 func (a *apiServer) KillNotebook(
 	ctx context.Context, req *apiv1.KillNotebookRequest,
 ) (resp *apiv1.KillNotebookResponse, err error) {
+	// CHECK: do we want to depend on "can view notebook" permission to kill it?
 	err = a.validateAndKillNotebook(ctx, req.NotebookId)
 	if err != nil {
 		return nil, err
 	}
+	// CHECK: when an NTSC is killed we also responds with the NTSC object.
+	// Is it okay to return this to a user who doesn't have view permission but has update permissions?
```

Comment:
sure, they just wont be able to actually see it right?

Reply:
correct. whether this is a concern or not sort of depends on what we do here https://github.com/determined-ai/determined/pull/5620#discussion_r1055810662

Reply:
cool, yeah I don't think this particular thing is concerning then